commit a300ce95e49fe079d4ca77370c23ce75db176218
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Fri Oct 8 12:28:02 2010 -0700

    Staging: brcm80211: sys: fix some comparison warnings
    
    Use min_t() instead of min() in some places to make the
    comparison explicit and resolve some compiler warnings.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/sys/wlc_channel.c
+++ b/drivers/staging/brcm80211/sys/wlc_channel.c
@@ -1376,7 +1376,7 @@ wlc_channel_reg_limits(wlc_cm_info_t *wl
 
 	/* Keep OFDM lmit below CCK limit */
 	if (BAND_2G(band->bandtype))
-		maxpwr = min(maxpwr, txpwr->cck[0]);
+		maxpwr = min_t(int, maxpwr, txpwr->cck[0]);
 
 	for (i = 0; i < WLC_NUM_RATES_OFDM; i++) {
 		txpwr->ofdm[i] = (u8) maxpwr;
--- a/drivers/staging/brcm80211/sys/wlc_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wlc_mac80211.c
@@ -4806,7 +4806,7 @@ wlc_doiovar(void *hdl, const bcm_iovar_t
 			bool override;
 
 			/* Remove override bit and clip to max qdbm value */
-			qdbm = (u8)min((int_val & ~WL_TXPWR_OVERRIDE), 0xff);
+			qdbm = (u8)min_t(u32, (int_val & ~WL_TXPWR_OVERRIDE), 0xff);
 			/* Extract override setting */
 			override = (int_val & WL_TXPWR_OVERRIDE) ? TRUE : FALSE;
 			err =
